src/mono/wasm/runtime/exports.ts

```diff
@@ -42,6 +42,8 @@ function initializeExports(globalObjects: GlobalObjects): RuntimeAPI {
     Object.assign(globals.internal, export_internal());
     Object.assign(runtimeHelpers, {
         stringify_as_error_with_stack: mono_wasm_stringify_as_error_with_stack,
+        set_runtime_aborted: set_runtime_aborted.bind(null, module),
```

Comment:
why are we doing `bind` here ?

Reply:
How else will they get `module`? It's a local